services-api/src/main/java/io/scalecube/services/MicroservicesContext.java

```diff
@@ -0,0 +1,35 @@
+package io.scalecube.services;
+
+import io.scalecube.net.Address;
+import io.scalecube.services.discovery.api.ServiceDiscovery;
+
+/**
+ * Context of Scale Cube node. Used in {@link ServiceFactory}.
+ *
+ * @see ServiceFactory
+ */
+public interface MicroservicesContext {
+
+  /**
+   * Id of Scale Cube node.
```

Comment:
Turn `Scale Cube` or `ScaleCube` to `Scalecube` evrey in the comments and variable names